.github/workflows/deploy_to_github_io.yml

```diff
@@ -21,15 +23,24 @@ jobs:
           auto-update-conda: true
           python-version: 3.8
           activate-environment: ASFHyP3
-          environment-file: ASFHyP3/conda-env.yml
+          environment-file: ASFHyP3/environment.yml
+
+      - name: test build of website
+        if: github.ref != 'refs/heads/main'
+        shell: bash -l {0}
+        run: |
+          cd ASFHyP3
+          mkdocs build
```

Comment:
my gut says this should be two workflows, one to test the build when the PR is opened, and a second to deploy when the PR is merged

Reply:
It could be -- will be rather repetitious though.